graph/index.html

```diff
@@ -171,7 +172,13 @@
   }
   pageInfra = infra;
 
-  const updatesUrl = `https://raw-updates.coreos.${infraPrefix}fedoraproject.org/v1/graph?basearch=${basearch}&stream=${stream}`;
+  var oci = urlParams.get('oci');
+  if (oci === null) {
+    oci = false;
+  }
+  pageOci = oci;
```

Comment:
The declaration of `pageOci` looks like unneeded since we don't use the var.

Reply:
good catch !